README.md

```diff
@@ -46,17 +47,17 @@ go build grofer.go
 
 ### Docker
 
-1. Get docker image using
-   ```
+1.	Get docker image using
+
+```
    docker pull ghcr.io/pesos/grofer
-   ```
+```
+
+Note: This will use the `latest` image which is built from the main branch. For stable releases, you will need to use the version number as the tag. Example: `docker pull ghcr.io/pesos/grofer:1.3.0`
 
-   Note: This will use the `latest` image which is built from the main branch.
-   For stable releases, you will need to use the version number as the tag.
-   Example: `docker pull ghcr.io/pesos/grofer:1.3.0`
+1.	Run using
 
-1. Run using
-   ```bash
+```bash
    docker run \
```

Comment:
Is there any specific reason for these formatting changes, or was it an auto-formatter?

It breaks the numbering ([see here](https://github.com/pesos/grofer/tree/actions#docker)):
![image](https://user-images.githubusercontent.com/34161949/130352218-5e6898de-f968-4295-8485-7a3d0a74f0d6.png)

Reply:
Oops, that's an auto formatter. That's on me. Will fix right away.